resources/css/bem/forum-topic-entry.less

```diff
@@ -4,27 +4,37 @@
 .forum-topic-entry {
   @_top: forum-topic-entry;
 
+  --forum-item-background-color: @osu-colour-b4;
+  --forum-item-background-color-hover: @osu-colour-b3;
+
   font-size: @font-size--normal;
   display: flex;
   margin: 2px -@forum-item-overflow;
-  background-color: @forum-item-background-color;
+  background-color: var(--forum-item-background-color);
   .default-border-radius();
   padding: 7px 0 7px @forum-item-overflow;
   color: @osu-colour-c2;
   transition: @forum-item-animate;
   position: relative;
 
   &:hover {
-    background-color: @forum-item-background-color-hover;
+    background-color: var(--forum-item-background-color-hover);
   }
 
   @media @desktop {
     margin-left: -@forum-item-overflow-desktop;
     margin-right: -@forum-item-overflow-desktop;
     padding-left: @forum-item-overflow-desktop;
   }
+  
+  &--old {
+    opacity: 0.7;
+  }
 
   &--deleted {
+    --forum-item-background-color: hsl(var(--colour-red-hue), 40%, 30%);
```

Comment:
I only meant the `@osu-colour-x` lesscss variable being deprecated, not the whole `--hsl-x`...